cmd/connector_ethereum.go

```diff
@@ -68,58 +50,17 @@ func connectChain(
 
 func initializeExtensions(
 	ctx context.Context,
-	config config.Extensions,
-	ethereumChain *ethereum.Chain,
+	tbtcEthereumChain chain.TBTCHandle,
 ) {
-	if len(config.TBTC.TBTCSystem) > 0 {
-		tbtcEthereumChain, err := ethereum.WithTBTCExtension(
-			ethereumChain,
-			config.TBTC.TBTCSystem,
-		)
-		if err != nil {
-			logger.Errorf(
-				"could not initialize tbtc chain extension: [%v]",
-				err,
-			)
-			return
-		}
-
+	if tbtcEthereumChain != nil {
 		tbtc.Initialize(ctx, tbtcEthereumChain)
+	} else {
+		logger.Errorf(
+			"could not initialize tbtc chain extension",
```

Comment:
I kept this as-is mostly because I'm not sure of the failure scenarios that would yield an error here. Agree that something more actionable would be 👌👌👌